lib/std/child_process.zig

```diff
@@ -99,12 +99,20 @@ pub const ChildProcess = struct {
                         return null;
                     }
                 },
+                .windows => {
+                    if (rus.rusage) |ru| {
+                        return ru.PeakWorkingSetSize;
+                    } else {
+                        return null;
+                    }
+                },
                 else => return null,
             }
         }
 
         const rusage_init = switch (builtin.os.tag) {
             .linux => @as(?std.os.rusage, null),
+            .windows => @as(?windows.PROCESS_MEMORY_COUNTERS, null),
```

Comment:
```suggestion
            .windows => @as(?windows.VM_COUNTERS, null),
```

(see https://github.com/ziglang/zig/pull/15035/files#r1145603999)